conda/recipes/raft-dask/meta.yaml

```
@@ -27,9 +27,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     {% if cuda_major == "11" %}
-    - {{ compiler('cuda11') }} ={{ cuda_version }}
-    {% else %}
-    - {{ compiler('cuda') }}
+    - {{ compiler('cuda11') }}
     {% endif %}
```

Think we want this logic

```suggestion
    {% if cuda_major == "11" %}
    - {{ compiler('cuda11') }} ={{ cuda_version }}
    {% else %}
    - {{ compiler('cuda11') }}
    {% endif %}
```